include/swift/Subsystems.h

```diff
@@ -153,6 +153,11 @@ namespace swift {
   /// instrumentation that has a high runtime performance impact.
   void performPlaygroundTransform(SourceFile &SF, bool HighPerformance);
   
+  /// Once parsing, name-binding and the playground transform (if required)
```

Comment:
This still says the playground transform would be complete, but you now run it before.